Cats/src/app/cat-card/cat-card.component.html

```diff
@@ -0,0 +1,22 @@
+<mat-card class="example-card" appearance="outlined">
+    <mat-card-header>
+      <div mat-card-avatar class="example-header-image"></div>
+      <mat-card-title>{{cat.name}}</mat-card-title>
+      <mat-card-subtitle>{{cat.breed}}</mat-card-subtitle>
+    </mat-card-header>
+    <div class="img_container">
```

Comment:
Please use dash (-) for class names with more than one word instead of underscore ( \_ )
Пожалуйста используй тире (-) для имен класса где больше чем одно слово вместо нижнего подчеркивания ( \_ )